cjwkernel/pandas/types.py

```diff
@@ -469,15 +491,15 @@ def coerce(cls, value: mtypes.Message) -> I18nMessage:
                     raise ValueError(
                         "Message can only have a single source, got %s" % source
                     )
-                supported_source_keys = {"module_id", "library"}
+                supported_source_keys = {"module", "library"}
                 if len(source) == 1 and set(source.keys()) - supported_source_keys:
                     raise ValueError(
                         "Unknown message source kind %s. Supported kinds: %s."
                         % (source.keys(), supported_source_keys)
                     )
             else:
-                source = {}
-            return cls(value[0], value[1], source)
+                source = None
+            return cls(value[0], value[1], source or None)
```

Comment:
I don't understand the `source or None`.

Please assert above that `len(source) == 1`. And then I think you can nix the `or None` here, right? (Otherwise, please add a comment saying what it does.)